# main
close_enough: 0.1 ##0.1 default
target_dt: 0.04 # this (s) should be slower than or equals to main_iter_rate (Hz). Default 0.04
average_speed: 0.2
cell_size: 0.05
inflation_radius: 0.22
log_odds_thresh: 10 # this should be smaller than log_odds_cap
log_odds_cap: 20
main_iter_rate: 25 

# motion
verbose_motion: false
use_internal_odom: false
axle_track: 0.16
wheel_radius: 0.033
weight_odom_v: 0.95
weight_odom_w: 0.05
straight_thresh: 0.05
motion_iter_rate: 50

# move
enable_move: true
verbose_move: true
Kp_lin: 2.4
Ki_lin: 0.0
Kd_lin: 0.2
max_lin_vel: 0.15 # max is 0.22
max_lin_acc: 1.0 ##default 1
Kp_ang: 1.3
Ki_ang: 0.0
Kd_ang: 0.3
max_ang_vel: 2.80 # max is 2.84
max_ang_acc: 4 ##default 2
move_iter_rate: 25

#tuning
tune_mode: false
tune_lin: false ##if tune_mode is true, ensure that only tune_lin OR tune_ang is true. This is because we tune the PIDs one at a time 
tune_ang: false